mitodl/mit-open
# Merge `main` into nextjs branch

### What are the relevant tickets?
For https://github.com/mitodl/hq/issues/5395

### Description (What does it do?)
Merges `main` into `nextjs` and fixes a few issues that arose.

### Reviewing:
**Suggestion:** Most of the commits on this branch are already on `main`. The merge conflicts were resolved in https://github.com/mitodl/mit-open/pull/1655/commits/c8696de47e51e99dc7f458fbcd0ee4740db3698b.

Some things were broken after the conflict resolutions, so I fixed them in the followup commit.

Most of the changes I made in the followup commit are related to making sure the header and footer look **identical** on  https://learn.mit.edu/ vs the nextjs branch. See video below, which switches between two browser tabs.

https://github.com/user-attachments/assets/55c09ced-6d44-4802-900c-1a522ff3d04c

### How can this be tested?
Run locally. You should get recent changes from `main`. Notably:

- header should look like learn.mit.edu
- Ecommerce page at `/cart`.
    - This is likely to be removed from MIT Learn and developed in a separate project
    - Accessing `/cart` requires setting a posthog flag. See https://github.com/mitodl/mit-open/pull/1634
